tests/baselines/reference/declarationMapsGeneratedMapsEnableMapping3.baseline.jsonc

```diff
@@ -0,0 +1,100 @@
+// === goToDefinition ===
+// === /index.ts ===
+// export class Foo {
+//     member: string;
+//     [|methodName|](propName: SomeType): SomeType { return propName; }
+//     otherMethod() {
+//         if (Math.random() > 0.5) {
+//             return {x: 42};
+//         }
+//         return {y: "yes"};
+//     }
+// }
+// 
+// export interface SomeType {
+//     member: number;
+// }
+
+// === /mymodule.ts ===
+// import * as mod from "/dist/index";
+// const instance = new mod.Foo();
+// instance./*GOTO DEF*/methodName({member: 12});
+
+// === TextSpan ===
+// === /mymodule.ts ===
+// import * as mod from "/dist/index";
+// const instance = new mod.Foo();
+// instance./*GOTO DEF*/[|methodName|]({member: 12});
+
+
+
+// === goToType ===
+// === /index.ts ===
+// export class Foo {
+//     member: string;
+//     methodName(propName: SomeType): SomeType { return propName; }
+//     otherMethod() {
+//         if (Math.random() > 0.5) {
+//             return {x: 42};
+//         }
+//         return {y: "yes"};
+//     }
+// }
+// 
+// export interface [|SomeType|] {
+//     member: number;
+// }
+
+// === /mymodule.ts ===
+// import * as mod from "/dist/index";
+// const instance = new mod.Foo();
+// instance./*GOTO TYPE*/methodName({member: 12});
+
+
+
+// === getDefinitionAtPosition ===
+// === /index.ts ===
+// export class Foo {
+//     member: string;
+//     [|methodName|](propName: SomeType): SomeType { return propName; }
+//     otherMethod() {
+//         if (Math.random() > 0.5) {
+//             return {x: 42};
+//         }
+//         return {y: "yes"};
+//     }
+// }
+// 
+// export interface SomeType {
+//     member: number;
+// }
+
+// === /mymodule.ts ===
+// import * as mod from "/dist/index";
+// const instance = new mod.Foo();
+// instance./*GOTO DEF POS*/methodName({member: 12});
+
+
+
+// === goToImplementation ===
+// === /index.ts ===
+// export class Foo {
+//     member: string;
+//     [|{| displayParts: [] |}methodName|](propName: SomeType): SomeType { return propName; }
```

Comment:
I don’t know if `displayParts` are even supposed to be an empty array here, but we should never render them inline.